sdk/documenttranslation/Azure.AI.DocumentTranslation/tests/Infrastructure/DocumentTranslationTestEnvironment.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+
+namespace Azure.AI.DocumentTranslation.Tests
+{
+    public class DocumentTranslationTestEnvironment : TestEnvironment
+    {
+        public DocumentTranslationTestEnvironment()
+        {
+        }
+
+        /// <summary>The name of the environment variable from which the Document Translator resource's endpoint will be extracted for the live tests.</summary>
+        internal const string EndpointEnvironmentVariableName = "DOCUMENT_TRANSLATION_ENDPOINT";
+
+        /// <summary>The name of the environment variable from which the Document Translator resource's API key will be extracted for the live tests.</summary>
+        internal const string ApiKeyEnvironmentVariableName = "DOCUMENT_TRANSLATION_API_KEY";
```

Comment:
If I haven't missed anything, I think we can make these `private`, but I don't feel strongly about it.